### DISCUSSION TITLE
70243EBD-3F8C-4A11-8587-961A8CB27F58

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#161] [General]
# Kepler-250

Kepler-250, also known as KOI-906, is 2535.22 light-years away from Earth. As of now, Kepler-250 has been confirmed to have three planets (one super Earth and two mini Neptune)

https://celestia.mobi/addon?item=70243EBD-3F8C-4A11-8587-961A8CB27F58

**Comment by exocritic5457** [comment]
What are the orbits in the picture supposed to indicate?

**Reply by kmjordan122313-lgtm** [reply]
agree, the orbits are confusing.

**Comment by kmjordan122313-lgtm** [comment]
this addon is good, but i'm confused about the orbits 😅
